packages/api/AlvTimeWebApi/Controllers/Admin/UserController.cs

```diff
@@ -30,18 +30,19 @@ public async Task<ActionResult<IEnumerable<UserResponseDto>>> FetchUsers()
         var users = await _userRepository.GetUsers(new UserQuerySearch());
         return Ok(users);
     }
-
+    
     [HttpPost("Users")]
     [AuthorizeAdmin]
-    public async Task<ActionResult<IEnumerable<UserResponseDto>>> CreateNewUsers([FromBody] IEnumerable<UserDto> usersToBeCreated)
+    public async Task<ActionResult<IEnumerable<UserResponseDto>>> CreateNewUsers([FromBody] IEnumerable<UserCreateRequest> usersToBeCreated)
     {
-        List<UserResponseDto> response = new List<UserResponseDto>();
+        var response = new List<UserResponseDto>();
         foreach (var user in usersToBeCreated)
         {
             response.Add(await _userService.CreateUser(new UserDto
```

Comment:
Should there be a try/catch when adding the users and then have return a json-object containing an array of successfull-users and an array of failed users? Just a suggestion

Reply:
I agree, that's a case that should be handled! Next round :)